doc/nrf/ug_radio_fem.rst

```diff
@@ -46,6 +46,7 @@ Adding support for nRF21540 in GPIO mode
 ****************************************
 
 The nRF21540 device is a range extender that can be used with the nRF52 and nRF53 Series devices.
+However, the software driver for the nRF21540 device in NCS is available onfy for nRF52 Series devices.
```

Comment:
```suggestion
However, the software driver for the nRF21540 device in NCS is available only for nRF52 Series devices.
```

Reply:
@ru-fu can you fix this typo in your new commit?